tools/clang/unittests/HLSLExec/CMakeLists.txt

```diff
@@ -25,7 +25,7 @@ include_directories(${DIASDK_INCLUDE_DIRS})
 include_directories(${D3D12_INCLUDE_DIRS})
 include_directories(${LLVM_MAIN_INCLUDE_DIR}/dxc/Test)
 
-add_dependencies(ExecHLSLTests dxcompiler)
+add_dependencies(ExecHLSLTests dxcompiler LLVMDxcSupport)
```

Comment:
Is this change still necessary, now that CoGetMalloc has been removed?

Reply:
It is not. Good catch! Changes reverted.